core/tst/software/aws/toolkits/core/region/AwsRegionTest.kt

```diff
@@ -40,3 +42,12 @@ class AwsRegionTest(private val region: AwsRegion, private val expectedCategory:
         assertThat(region.category, equalTo(expectedCategory))
     }
 }
+
+fun anAwsRegion(id: String = aRegionId(), name: String = aString(), partitionId: String = aString()) = AwsRegion(id, name, partitionId)
```

Comment:
Shouldn't this also go in credential mocks? (or its own mocks file?)

Reply:
They're not really mock's they're just test data generators